tests/foreman/ui/test_org.py

```diff
@@ -308,6 +307,31 @@ def test_positive_delete_1(self, org_name):
             self.org.remove(org_name, really=True)
             self.assertIsNone(self.org.search(org_name))
 
+    @skip_if_bug_open('bugzilla', 1225588)
+    @data(*generate_strings_list())
+    def test_positive_delete_bz1225588(self, org_name):
+        """@test: Create Organization with valid values and upload manifest.
+        Then try to delete the organization.
+
+        @feature: Organization Positive Delete Test.
+
+        @assert: Organization is deleted.
+
+        """
+        org = entities.Organization(name=org_name).create_json()
+        manifest_path = manifests.clone()
+        entities.Organization(id=org['id']).upload_manifest(path=manifest_path)
```

Comment:
You can call `create()` instead of `create_json()`. And these method calls can be chained.

```python
entities.Organziation(name=org_name).create().upload_manifest(path=manifests.clone())
```